presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/file/PartitionMetadata.java

```diff
@@ -198,6 +199,7 @@ public Partition toPartition(String databaseName, String tableName, List<String>
                 Optional.empty(),
                 eligibleToIgnore,
                 sealedPartition,
-                0);
+                0,
+                new DateTime(-1000));
```

Comment:
Same there. Could we use 0 as the default value and use int type?

Reply:
Yeah, we can. I use -1000 since DEPA wants the default value to be -1, and I want to be consistent with that. Using DateTime is mainly consistent with the variable types in QueryStats. I agree that integer should be more efficient.